Week01/Problem02/ademissie_02.py

```diff
@@ -0,0 +1,8 @@
+variable1 = 1
+variable2 = 1
+sum = 0
```

Comment:
you may consider using a different name for this variable, as `sum` is a built in function that you have implicitly overwritten